lib/config-generator.js

```diff
@@ -67,8 +68,13 @@ class ConfigGenerator {
 
         config.stats = this.buildStatsConfig();
 
+        let extensions = ['.js', '.jsx', '.ts', '.tsx'];
+        if (this.webpackConfig.useTypeScriptLoader) {
+            extensions.push('.json');
+        }
```

Comment:
Odd!... I recall having issues with loader not resolving the `tsconfig.json` file for TypeScript compilation if not added the `json` extension. But now I see on my project everything is fine without it and also the tests are fine. So I'll remove this.